File: src/server/BuildingBlock/MovieTicket.Domain/Entities/SeatReservation.cs
using MovieTicket.Core.Domain;

namespace MovieTicket.Domain.Entities;

public class SeatReservation : EntityBase
{
    public Guid? SeatId { get; set; }
    public Guid? ReservationId { get; set; }
    
    public virtual Reservation Reservation { get; set; }
    
    public virtual Seat Seat { get; set; }
}

File: src/client/mobile/app/src/main/java/com/superman/movieticket/domain/entities/SeatReservation.kt
package com.superman.movieticket.domain.entities

import com.superman.movieticket.core.entity.BaseEntity
import java.io.Serializable

data class SeatReservation (
    var seatId: String,
    var seat: Seat?,
    var reservationId: String,
    var reservation: Reservation?
) : BaseEntity(),Serializable